### PR TITLE
Update from auto-update/renku-core-2.8.0

### PR DESCRIPTION
**Pull request by RenkuBot**
This is an automated pull request.

/deploy extra-values=global.renku.cli_version=2.8.0rc3

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -15,6 +15,12 @@ User-Facing Changes
   (`#1666 <https://github.com/SwissDataScienceCenter/renku-graph/issues/1666>`_).
 - **KG**: A new `GET /knowledge-graph/version` API.
   (`#1760 <https://github.com/SwissDataScienceCenter/renku-graph/pull/1760>`_).
+- **Core Service, CLI**: Add support for specifying project images.
+  (`#3623 <https://github.com/SwissDataScienceCenter/renku-python/issues/3623>`)
```
Comment (lokijuhy) [inline CHANGELOG.rst:19]:
```suggestion
- **Core Service, CLI**: Add support for specifying a project image during project initialization.
  (`#3623 <https://github.com/SwissDataScienceCenter/renku-python/issues/3623>`)
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -15,6 +15,12 @@ User-Facing Changes
...
+- **CLI**: Add support for pausing/resuming sessions.
+  (`#3633 <https://github.com/SwissDataScienceCenter/renku-python/issues/3633>`)
+- **Core Service**: Add prometheus metrics.
+  (`#3640 <https://github.com/SwissDataScienceCenter/renku-python/issues/3640>`)
```
Comment (lokijuhy) [inline CHANGELOG.rst:23]: Please move this to the "Internal Changes" section

**Review by lokijuhy — DISMISSED** [review]
The Changelog items need to be moved to 0.44.0, too

File: CHANGELOG.rst [hunk 3]
```diff
@@ -1,9 +1,48 @@
...
+on both the CLI and the UI.
+
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
+
+**✨ Improvements**
+
+- **Core Service, CLI**: Add support for specifying a project image during
+  project initialization
+  (`#3623 <https://github.com/SwissDataScienceCenter/renku-python/issues/3623>`).
```
Comment (leafty) [inline CHANGELOG.rst:16]:
```suggestion
  (`#3623 <https://github.com/SwissDataScienceCenter/renku-python/issues/3623>`_).
```

File: CHANGELOG.rst [hunk 4]
```diff
@@ -1,9 +1,48 @@
...
+- **CLI**: Add support for pausing & resuming remote sessions from the cli
+  (`#3633 <https://github.com/SwissDataScienceCenter/renku-python/issues/3633>`).
```
Comment (leafty) [inline CHANGELOG.rst:18]:
```suggestion
  (`#3633 <https://github.com/SwissDataScienceCenter/renku-python/issues/3633>`_).
```

File: CHANGELOG.rst [hunk 5]
```diff
@@ -1,9 +1,48 @@
...
+
+Internal Changes
+~~~~~~~~~~~~~~~~
+
+**✨ Improvements**
+
+- **Core Service, CLI**: Make slug and name consistent with rest of platform
+  (`#3620 <https://github.com/SwissDataScienceCenter/renku-python/issues/3620>`).
```
Comment (leafty) [inline CHANGELOG.rst:26]:
```suggestion
  (`#3620 <https://github.com/SwissDataScienceCenter/renku-python/issues/3620>`_).
```

File: CHANGELOG.rst [hunk 6]
```diff
@@ -1,9 +1,48 @@
...
+- **Core Service**: Add prometheus metrics
+  (`#3640 <https://github.com/SwissDataScienceCenter/renku-python/issues/3640>`).
```
Comment (leafty) [inline CHANGELOG.rst:28]:
```suggestion
  (`#3640 <https://github.com/SwissDataScienceCenter/renku-python/issues/3640>`_).
```

File: CHANGELOG.rst [hunk 7]
```diff
@@ -1,9 +1,48 @@
...
+- **UI**: Adapt dataset APIs to the new naming convention used in the backend
+  (`#2854 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2854>`).
```
Comment (leafty) [inline CHANGELOG.rst:30]:
```suggestion
  (`#2854 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2854>`_).
```

File: CHANGELOG.rst [hunk 8]
```diff
@@ -1,9 +1,48 @@
...
+
+Individual components
+~~~~~~~~~~~~~~~~~~~~~
+
+- `renku-ui 3.17.1 <https://github.com/SwissDataScienceCenter/renku-ui/releases/tag/3.17.1>`_
+- `renku-python 2.8.0 <https://github.com/SwissDataScienceCenter/renku-python/tree/v2.8.0>`
```
Comment (leafty) [inline CHANGELOG.rst:36]:
```suggestion
- `renku-python 2.8.0 <https://github.com/SwissDataScienceCenter/renku-python/tree/v2.8.0>`_
```

**Review by leafty — COMMENTED** [review]
🔗  are broken!

File: CHANGELOG.rst (hunk 3 above)
Comment (lorenzo-cavazzi) [reply CHANGELOG.rst]: Ooops :speak_no_evil:  Thank you for spotting those!

**Review by bethcg — APPROVED** [review]

**Pull request merged**